app/src/main/java/com/github/libretube/ui/preferences/MainSettings.kt

```diff
@@ -98,7 +102,11 @@ class MainSettings : BasePreferenceFragment() {
 
                 if (BuildConfig.VERSION_NAME != updateInfo.name) {
                     // show the UpdateAvailableDialog if there's an update available
-                    UpdateAvailableDialog(updateInfo).show(
+                    val encodedUpdateInfo = Json.encodeToString(updateInfo)
+                    val newUpdateAvailableDialog = UpdateAvailableDialog()
+                    newUpdateAvailableDialog.arguments =
+                        bundleOf(IntentData.updateInfo to encodedUpdateInfo)
```

Comment:
See the earlier comment.

```suggestion
                    val newUpdateAvailableDialog = UpdateAvailableDialog()
                    newUpdateAvailableDialog.arguments =
                        bundleOf(IntentData.updateInfo to updateInfo)
```